package.json

```diff
@@ -54,8 +54,10 @@
     "lodash.groupby": "^4.6.0",
     "lodash.times": "^4.3.2",
     "node-env-flag": "^0.1.0",
+    "node-pg-migrate": "^6.2.2",
```

Comment:
Even though we don't expect to change our DB schema super often, I think its worth having a migration package to make it easier to evolve this over time. Ideally we should also look at running `npm run migrate up` on deploy. For the first deploy I think it can live with doing it manually.

Reply:
Does this need to be a prod dep or can it still be utilized as a dev dep?

Reply:
In order to create the DB (in the first instance) or modify it (later) we need to be able to run `npm run migrate up` in the production environment so it has to be a prod dependency.